docs/sources/datasources/elasticsearch.md

```diff
@@ -80,6 +80,13 @@ number followed by a valid time identifier, e.g. `1m` (1 minute) or `30s` (30 se
 | `s`        | second      |
 | `ms`       | millisecond |
 
+### X-Pack Enabled
```

Comment:
Why are the two headings here in title case? If you are not matching the UI, then headings should be in sentence case.

Reply:
It is matching the UI, but probably we should change the UI to be sentence case?

Reply:
@Elfo404 - yes, please! TY

Reply:
@achatterjee-grafana @oddlittlebird done :) thanks for the feedback :)